cmd/snap-bootstrap/cmd_initramfs_mounts_test.go

```diff
@@ -3310,7 +3310,7 @@ grade=signed
 		c.Check(fiParent.Mode(), Equals, os.FileMode(os.ModeDir|0750))
 	}
 
-	c.Check(filepath.Join(ephemeralUbuntuData, "system-data/var/lib/snapd/state.json"), testutil.FileEquals, `{"data":{"auth":{"last-id":1,"macaroon-key":"not-a-cookie","users":[{"id":1,"name":"mvo"}]}},"changes":{},"tasks":{},"last-change-id":0,"last-task-id":0,"last-lane-id":0,"last-notice-id":0}`)
+	c.Check(filepath.Join(ephemeralUbuntuData, "system-data/var/lib/snapd/state.json"), testutil.FileEquals, `{"data":{"auth":{"last-id":1,"macaroon-key":"not-a-cookie","users":[{"id":1,"name":"mvo"}]}},"changes":{},"tasks":{},"last-change-id":0,"last-task-id":0,"last-lane-id":0,"last-notice-id":0,"notice-last-date":"0001-01-01T00:00:00Z"}`)
```

Comment:
I believe this should be `"notice-last-timestamp"` now. Also, if golang json encodings of times are nanosecond precision, shouldn't the test string include nanosecond precision here as well?

Reply:
Ops! I was filtering `*test.go` files, so I didn't find them. Sorry. Fixing all!